metsim/metsim.py

```diff
@@ -247,6 +249,18 @@ def met_data(self):
             self._met_data['elev'] = self.domain['elev']
             self._met_data['lat'] = self.domain['lat']
             self._met_data['lon'] = self.domain['lon']
+
+            # process constant_vars
+            constant_vars = self.params.get('constant_vars', None)
+            if constant_vars:
+                if self.params['forcing_fmt'] != 'binary':
+                    template_var = list(self.params.get('forcing_vars', None).values())[0]
```

Comment:
I think it would be clearer, and eliminate the `if` statement to get the template variable by doing something like `template_var = self._met_data[list(self._met_data)[0]]`

Reply:
Done